policies/travel-policy.md

```diff
@@ -132,28 +131,28 @@ are inclusive of 1 checked piece of luggage per person.
     flight class is at the Traveller's own expense.
 -   Additional baggage and/or overweight baggage fees are at the
     Traveller's own cost.
-- 	Airport parking is also possible utilize
+-   Airport parking is also reimbursable.
 
 ##### Flight Class Table
  Total Flying Time is the time from the scheduled departure of the first flight to the scheduled arrival of the last flight
- 
- | Total Flying Time | Flight Class | Where Flight Class is not available |
- | --- | --- | ---|
- | Less than 6 hours | Economy| n/a |
- | Greater than 6 but less than 12 hours | Premium Economy | Economy with an extra pre-arrival day (i.e. arrives 2 days prior) |
- | Greater than 12 hours | Business Class | Premium Economy or Economy with an extra pre-arrival day (i.e. arrives 2 days prior) |
+
+ | Total Flying Time | Flight Class |
+ | --- | --- |
+ | Less than 6 hours | Economy|
+ | Greater than 6 but less than 12 hours | Premium Economy |
+ | Greater than 12 hours | Business Class |
 
 #### Rental Cars
 
 ***Rental*** Cars can be rented when, there is either no other means
 of transportation available or where it is either more costly or
 impractical to use other transportation.
 
--   Any additional insurance or add on items e.g. GPS devices and any
-    fines or excesses incurred whilst using a rental car will be at the
-    cost of the Traveller.
+-   Any additional insurance or add on items e.g. any fines or excesses
+    incurred whilst using a rental car will be at the cost of the Traveller.
 -   Tolls &/or Parking fees will be eligible for reimbursement.
 -   Excess reduction fees are reimbursable.
```

Comment:
Possibly need to loosen the wording here.  I've rented cars where the _full value of the car_ was my liability unless I paid extra.  This wasn't an an _excess reduction fee_ but was essentially mandatory.

Reply:
Do you want to give me some words around what you think would cover the scenario?
Thanks Tam

Reply:
This scenario would be covered by purchasing the Excess reduction at the time of renting and the excess reduction fee is reimbursable.  So no change to the wording is needed.